oaknational/Oak-Web-Application
# Fix/unit listing developmentstage ordering

## Description

Music year: 2002

- List of changes

Sorts the developmentstages array on the unit listing page into the correct order
https://www.notion.so/oaknationalacademy/Development-stages-out-of-order-on-unit-listing-page-tabs-2ec8e4c1c7e74eb79f6d9e7401355d4a?pvs=4 

## How to test

1. Go to https://deploy-preview-2313--oak-web-application.netlify.thenational.academy
3. You should see no changes

## Screenshots

How it used to look:
<img width="500" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/433dd3e8-0ce3-468b-be25-9cab6f2c8979">

How it should now look:

<img width="500" alt="Screenshot 2024-03-18 at 13 46 25" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/95860c2b-e411-4cc2-bab0-106df9361208">
